.grabl/automation.yml

```diff
@@ -33,40 +33,124 @@ build:
        bazel run //:typedb-extractor -- dist/typedb-all-linux
        sudo systemd-run ./dist/typedb-all-linux/typedb server
        sleep 60
-       bazel test //catalogue_of_life:tests --test_output=errors
+       bazel test //biology/catalogue_of_life:tests --test_output=errors
```

Comment:
Here and in bazel files most changes are simply updating paths after the restructure.